backend/backend.go

```diff
@@ -638,6 +644,12 @@ func (backend *Backend) Start() <-chan interface{} {
 	backend.configureHistoryExchangeRates()
 
 	backend.environment.OnAuthSettingChanged(backend.config.AppConfig().Backend.Authentication)
+
+	if backend.testing {
```

Comment:
tiny nit, but shouldn't this be `backendConfig.AppConfig().Backend.StartInTestnet`? The config does not need to be touched if testing is true due to the `-testnet` flag.